docs/source/set_dev_env_with_vagrant.rst

```diff
@@ -8,15 +8,31 @@ development environment, all you need is just to run
 
     vagrant up --provider virtualbox
 
+Note if you're under proxy, then you need to modify Vagrantfile first by adding proxy
+
+::
+
+    config.proxy.http = "http://;proxy>:<port>"
+    config.proxy.https = "http://<proxy>:<port>"
+
 After ``vagrant`` succeeds to run the virtual machine, you will get a complete
 environment to develop Nitrate,
 
 * a Python virtual environment creatd at ``$HOME/nitrate-env/`` with all
   necessary dependecies installed.
+ 
+* install all required Python dependencies
+
+  ::
+
+    cd /code
+    sudo pip install -r requirements/base.txt
+    sudo pip install -r requirements/devel.txt
```

Comment:
I guess you did not read through the Vagrantfile. There is one step `devenv` which sets up a Python virtual environment and all dependent packages are installed from there.

Reply:
I'm just a dumb user, I don't want to read config files, I want to follow instructions and get the tool working. If it's not in the doc - nobody will know about it.